### DISCUSSION TITLE
Coding import pip/extension

### DISCUSSION THREAD

**Discussion by SKumulia** [teamalgoritma/community#14] [Q&A] [answered]
Apakah ekstension harus selalu di import setiap kali akan menuliskan kode pada environment yg sama?

**Comment by inytss** [comment, marked as answer]
Untuk package/library ketika membuka sebuah notebook baru walaupun di environment yang sama maka wajib harus mengimport library/package tersebut. Contoh:
`import pandas as pd`
Disarankan untuk import library, codenya diletakkan dipaling atas notebook.
